nimbus/src/dbcache.rs

```diff
@@ -43,14 +44,21 @@ impl DatabaseCache {
         // By passing in the active `writer` we read the state of enrollments
         // as written by the calling code, before it's committed to the db.
         let experiments = get_enrollments(&db, &writer)?;
+
         // Build the new hashmap.
         let mut eb = HashMap::with_capacity(experiments.len());
```

Comment:
I like the renaming you did, but I think this var should be renamed too (`eb` made sense when `experiment_branches` existed, but not-so-much now)

Reply:
Done.